docs/06-quality-control.md

```diff
@@ -1,3 +1,38 @@
 # Quality control
 
 When processing data, it is considered good practice (and is also very useful!) to inspect the intermediate output files/data created after every few steps. Here are some resources that can guide you through this process.
+
+###Let's start with introduction:
+
+
+Quality Checks for fMRI Data - Lecture on quality control for neuroimaging data, especially fMRI data. 
+##Quality checks include:
+
+
+1. Looking for susceptibility and pathological artifacts
+
+2. Checking whether motion exceeds your lab's thresholds
+
+3. Keeping records to make sure you have the data that you should have. 
+ URL: https://www.youtube.com/watch?v=fvv2dr3pT7I ) 
```

Comment:
I suggest you to use Markdown syntax like like
```suggestion
 [URL](https://www.youtube.com/watch?v=fvv2dr3pT7I) 
```
So `[URL](https://www.youtube.com/watch?v=fvv2dr3pT7I)` will render [URL](https://www.youtube.com/watch?v=fvv2dr3pT7I) :)

Reply:
I corrected it in the last commit 11 minutes ago, I will delete old commits